docs/reference-guides/block-api/block-templates.md

```diff
@@ -140,6 +139,25 @@ _Options:_
 
 You can use this with `templateLock` to lock all blocks except a single block by using `false` in `remove` or `move`.
 
+```php
+$template = array(
+	array( 'core/image', array(
+		'align' => 'left',
+	) ),
+	array( 'core/heading', array(
+		'placeholder' => 'Add Author...',
+	) ),
+	// Make a Paragraph block optional.
```

Comment:
Rather than using the word "optional", I might describe it as _Allow a Paragraph block to be moved or removed_. It might seem too explicit but I think it can help in these kinds of docs to be very clear with what's going on at a glance.

Reply:
Updated. Thanks, @annezazu.